### ISSUE TITLE
Incorrect error with punctuation in string

### ISSUE THREAD

**Issue by garyo** [godotengine/godot-vscode-plugin#64]
This statement:
```
			info("Mesh names match, deleting old one")
```
gives the error
```
 ': or \' expected at end of line.
```
But there is no error. If you delete the comma in the string, the error message goes away.

**Comment by Geequlim** [comment]
Related #31 #63

**Comment by Geequlim** [comment]
The new version fixed the problem and will be published when godot 3.2 RC release.

**Comment by Geequlim** [comment]
I'm closing this as the new version has fixed this issue.
You can try the new version of the plugin with godot 3.2 from the release page
